PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -8,20 +10,38 @@ public static void Main(string[] args)
         Question[] questions = LoadQuestions(filePath);
 
         int numberCorrect = 0;
+        int numberIncorrect = 0;
         for (int i = 0; i < questions.Length; i++)
         {
             bool result = AskQuestion(questions[i]);
             if (result)
             {
                 numberCorrect++;
             }
+            else
+            {
+                numberIncorrect++;
+            }
         }
+        
+        
+        // added a message to let the user know that the trivia has been completed along with some additional stats
+        Console.WriteLine("\nYou have completed the trivia!");
+        Console.WriteLine("Correct answers: " + numberCorrect);
+        Console.WriteLine("Incorrect answers: " + numberIncorrect);
         Console.WriteLine("You got " + GetPercentCorrect(numberCorrect, questions.Length) + " correct");
+        
+        DisplayMotivationalMessage(numberCorrect, questions.Length);
     }
 
     public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
     {
-        return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+        /*
+         Casted the numerCorrectAnswers to a double value so we could a correct perentage value then casted
+            the result back to an int
+         */
+        
+        return (int)((double)numberCorrectAnswers / numberOfQuestions * 100) + "%";
```

Comment:
```suggestion
        return ((float)numberCorrectAnswers / numberOfQuestions * 100) + "%";
```
I think this would be a bit simpler instead of double casting - and the int seems unnecessary nonetheless